profiles/common/themes/ec_resp/css/interinstituinal_less/eu_resp.less

```diff
@@ -0,0 +1,107 @@
+/*!
+ * Eu resp.less
+ * less meta file
+ *
+ * specific stylesheets for IE are called in template.php
+ */
+ 
+//** Load fonts from this directory.
+
+// Core variables and mixins
+@import "../../bootstrap/less/variables.less";
+@icon-font-path:          "../bootstrap/fonts/";
+@import "../../bootstrap/less/mixins.less";
+
+// Reset
+@import "../../bootstrap/less/normalize.less";
+@import "../../bootstrap/less/print.less";
+
+// Core CSS
+@import "../../bootstrap/less/scaffolding.less";
+@import "../../bootstrap/less/type.less";
+@import "../../bootstrap/less/code.less";
+@import "../../bootstrap/less/grid.less";
+@import "../../bootstrap/less/tables.less";
+@import "../../bootstrap/less/forms.less";
+@import "../../bootstrap/less/buttons.less";
+
+// Components
+@import "../../bootstrap/less/component-animations.less";
+@import "../../bootstrap/less/glyphicons.less";
+@import "../../bootstrap/less/dropdowns.less";
+@import "../../bootstrap/less/button-groups.less";
```

Comment:
import ``ec_resp.less`` instead of manually include all less files

```
@import "../../bootstrap/less/ec_resp.less";

@import interinstitutional less
```